app/src/main/java/it/niedermann/nextcloud/deck/persistence/sync/adapters/db/DataBaseAdapter.java

```diff
@@ -659,10 +659,10 @@ public Board getBoardByLocalIdDirectly(long localId) {
         return db.getUserDao().searchUserByUidOrDisplayName(accountId, boardId, notYetAssignedToLocalCardId, "%" + searchTerm.trim() + "%");
     }
 
-    public LiveData<List<User>> searchUserByUidOrDisplayNameForACL(final long accountId, final long notYetAssignedToACL, final String searchTerm) {
-        validateSearchTerm(searchTerm);
-        return db.getUserDao().searchUserByUidOrDisplayNameForACL(accountId, notYetAssignedToACL, "%" + searchTerm.trim() + "%");
-    }
+//    public LiveData<List<User>> searchUserByUidOrDisplayNameForACL(final long accountId, final long notYetAssignedToACL, final String searchTerm) {
+//        validateSearchTerm(searchTerm);
+//        return db.getUserDao().searchUserByUidOrDisplayNameForACL(accountId, notYetAssignedToACL, "%" + searchTerm.trim() + "%");
+//    }
```

Comment:
If no longer needed: Why not remove?

Reply:
Because for no reason. Had it there, saw it wasn't used, commenting out didn't break anything, but wasn't my focus, too. Feel free to remove

Reply:
Oh, and also the method in the dao